packages/sanity/src/core/form/inputs/DateInputs/base/DateTimeInput.tsx

```diff
@@ -11,6 +11,7 @@ import {
   useState,
 } from 'react'
 import FocusLock from 'react-focus-lock'
+import {useTranslation} from 'react-i18next'
```

Comment:
this should be imported from the `useTranslation` hook defined in this package and not the `react-i18next` one